profile/isa_ro_crate.md

```diff
@@ -69,122 +73,120 @@ Protocol --reagent---> ont
+Is based upon [schema.org/Dataset](https://schema.org/Dataset) and maps to the [ISA-JSON Assay](https://isa-specs.readthedocs.io/en/latest/isajson.html#assay-schema-json)
 
 | Property | Required | Expected Type | Description |
 |----------|----------|---------------|-------------|
-|@type |MUST|Text|must be 'Dataset'|
+|@type |MUST|Text|must be '[schema.org/Dataset](https://schema.org/Dataset)'|
 |@id|MUST|Text or URL|Should be a subdirectory corresponding to this assay.|
 |additionalType|MUST|Text or URL|‘Assay’ or ontology term to identify it as an Assay|
-|creator|MUST|Person|The performer of the experiments.|
+|creator|MUST|[schema.org/Person](https://schema.org/Person)|The performer of the experiments.|
 |identifier|MUST|Text or URL|Identifying descriptor of the assay.|
 |headline|MUST|Text|A title of the assay.|
-|_processSequence_|MUST|LabProcess|The experimental processes performed in this assay.|
-|measurementMethod|MUST|URL or DefinedType|Describes the type measurement e.g Complexomics or transcriptomics as an ontology term|
-|measurementTechnique|MUST|URL or DefinedType|Describes the type of technology used to take the measurement, e.g mass spectrometry or deep sequencing|
+|_processSequence_|MUST|[bioschemas.org/LabProcess](https://bioschemas.org/LabProcess)|The experimental processes performed in this assay.|
+|measurementMethod|MUST|URL or [schema.org/DefinedTerm](https://schema.org/DefinedTerm)|Describes the type measurement e.g Complexomics or transcriptomics as an ontology term|
+|measurementTechnique|MUST|URL or [schema.org/DefinedTerm](https://schema.org/DefinedTerm)|Describes the type of technology used to take the measurement, e.g mass spectrometry or deep sequencing|
 |hasPart|SHOULD|File|The data files resulting from the process sequence|
 |description|SHOULD|Text|A short description of the assay (e.g. an abstract)|
-|variableMeasured|COULD|Text or PropertyValue|The target variable being measured E.g protein concentration|
+|variableMeasured|COULD|Text or [schema.org/PropertyValue](https://schema.org/PropertyValue)|The target variable being measured E.g protein concentration|
 |dateCreated|SHOULD|DateTime|When the Assay was created|
 |dateModified|SHOULD|DateTime|When the Assay was last modified|
-|citation|COULD|ScholarlyArticle|A publication corresponding to this assay.|
-|comment|COULD|Comment|Comment|
+|citation|COULD|[schema.org/ScholarlyArticle](https://schema.org/ScholarlyArticle)|A publication corresponding to this assay.|
+|comment|COULD|[schema.org/Comment](https://schema.org/Comment)|Comment|
 
 
 ### LabProcess (_new type_)
 
-Has the new suggested Bioschemas LabProcess type and maps to the [ISA-JSON Process](https://isa-specs.readthedocs.io/en/latest/isajson.html#process-schema-json)
+Has the new suggested Bioschemas [bioschemas.org/LabProcess](https://bioschemas.org/LabProcess) type and maps to the [ISA-JSON Process](https://isa-specs.readthedocs.io/en/latest/isajson.html#process-schema-json)
```

Comment:
Maybe change to `Has the Bioschemas DRAFT type and...`

Reply:
Not necessary, but the LabProcess is on the same level like LabProtocol and such now.